docs/docker-machine-drivers.md

```diff
@@ -55,6 +55,18 @@ $ newgrp libvirt
 <a name="xhyve-driver"></a>
 ## xhyve driver
 
+Minishift is currently tested against `docker-machine-driver-xhyve` 0.3.1
```

Comment:
The documentation part is being looked into as part of: https://github.com/minishift/minishift/issues/203 where the whole document will be reviewed in detail.